build/Dockerfile.alpine

```diff
@@ -0,0 +1,8 @@
+FROM alpine:latest
+
+RUN apk add --no-cache ca-certificates \
+    && rm -rf /var/cache/apk/*
+
+COPY nancy /usr/local/bin/nancy
+
+CMD ["nancy"]
```

Comment:
Shouldn't this be `ENTRYPOINT ["nancy"]`?

Reply:
Usage example, https://gitlab.com/greut/eclint/-/jobs/471748010